src/olympia/search/tests/test_views.py

```diff
@@ -622,21 +622,75 @@ def test_search_doesnt_return_unlisted_addons(self):
         assert addon.pk not in self.get_results(response)
 
     def test_webextension_boost(self):
-        web_extension = self.addons[1]
+        web_extension = list(sorted(
+            self.addons, key=lambda x: x.name, reverse=True))[0]
         web_extension.current_version.files.update(is_webextension=True)
         web_extension.save()
         self.refresh()
 
-        response = self.client.get(self.url, {'q': 'addon'})
+        response = self.client.get(self.url, {'q': 'Addon'})
         result = self.get_results(response, sort=False)
         assert result[0] != web_extension.pk
 
         create_switch('boost-webextensions-in-search')
         # The boost chosen should have made that addon the first one.
-        response = self.client.get(self.url, {'q': 'addon'})
+        response = self.client.get(self.url, {'q': 'Addon'})
         result = self.get_results(response, sort=False)
         assert result[0] == web_extension.pk
 
+    def test_score_boost_name_match(self):
+        addons = [
```

Comment:
huh, very fair point. Potentially the issue of random test failures that I saw in the past few days. I'll look into making the tests more explicit, maybe even moving them to a separate test-class to avoid any external influences.